components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1807,12 +1807,19 @@ where
         }
         self.ctx.pending_count += 1;
         self.ctx.has_ready = true;
+
+        // follower will clean it as soon as applied.
```

Comment:
Hi, @Connor1996 . I have a question, will the witness raft log gc part we do conflict with the oom prevention mechanism of this PR?

Reply:
this pr just clean the cache, not do the raft log gc